gemfiles/prod/Gemfile.lock

```diff
@@ -458,7 +463,7 @@ GEM
     net-http-digest_auth (1.4)
     net-http-persistent (2.9.4)
     netrc (0.11.0)
-    newrelic_rpm (5.6.0.349)
+    newrelic_rpm (5.7.0.350)
```

Comment:
mmmm... This happened after doing `make bundle` and I am not sure if it is a good idea to add this change in this PR... probably not.

Reply:
Well I think that is OK but better if it is another PR

Reply:
https://github.com/3scale/porta/pull/620 👍